Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -1,150 +1,157 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
+using System.Collections.Generic;
+using System.Linq;
 using Microsoft.MixedReality.Toolkit.Boundary;
+using Microsoft.MixedReality.Toolkit.CameraSystem;
 using Microsoft.MixedReality.Toolkit.Diagnostics;
 using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Rendering;
+using Microsoft.MixedReality.Toolkit.SceneSystem;
 using Microsoft.MixedReality.Toolkit.SpatialAwareness;
 using Microsoft.MixedReality.Toolkit.Teleport;
 using Microsoft.MixedReality.Toolkit.Utilities;
-using System;
-using System.Collections.Generic;
-using System.Linq;
 using Unity.Profiling;
 using UnityEngine;
 using UnityEngine.EventSystems;
-using Microsoft.MixedReality.Toolkit.SceneSystem;
-using Microsoft.MixedReality.Toolkit.CameraSystem;
-using Microsoft.MixedReality.Toolkit.Rendering;
-
 #if UNITY_EDITOR
 using Microsoft.MixedReality.Toolkit.Input.Editor;
 using UnityEditor;
+
 #endif
 
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// This class is responsible for coordinating the operation of the Mixed Reality Toolkit. It is the only Singleton in the entire project.
-    /// It provides a service registry for all active services that are used within a project as well as providing the active configuration profile for the project.
-    /// The Profile can be swapped out at any time to meet the needs of your project.
+    ///     This class is responsible for coordinating the operation of the Mixed Reality Toolkit. It is the only Singleton in
+    ///     the entire project.
+    ///     It provides a service registry for all active services that are used within a project as well as providing the
+    ///     active configuration profile for the project.
+    ///     The Profile can be swapped out at any time to meet the needs of your project.
     /// </summary>
     [DisallowMultipleComponent]
     [AddComponentMenu("Scripts/MRTK/Core/MixedRealityToolkit")]
     public class MixedRealityToolkit : MonoBehaviour, IMixedRealityServiceRegistrar
     {
-        private static bool isInitializing = false;
-        private static bool isApplicationQuitting = false;
-        private static bool internalShutdown = false;
-        private const string NoMRTKProfileErrorMessage = "No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit";
+        private static bool isInitializing;
+        private static bool isApplicationQuitting;
+        private static bool internalShutdown;
+
+        private const string NoMRTKProfileErrorMessage =
+            "No Mixed Reality Configuration Profile found, cannot initialize the Mixed Reality Toolkit";
 
         /// <summary>
-        /// Whether an active profile switching is currently in progress
+        ///     Whether an active profile switching is currently in progress
```

Comment:
Can you remove the formatting changes from this PR please? It makes the actual changes a bit harder to review.
Changes I've noticed:

1. Additional spaces at the beginning of `summary` blocks
2. Removal of `{ }` from short `if` statements
3. Removal of default values from variable initializers
4. Replacing explicit variable types with `var`

These don't fully match with the [coding conventions](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#use-braces) we've used in the MRTK.

Reply:
Thanks for a response! Sorry for messing up with your coding conventions.

I have applied your suggestion to most of the code. It may still not be perfect, but hopefully it'll be clearer now!

Reply:
Thanks!